### PR TITLE
[CephAdm][Tier2]Configure RGW in warn state

### PR DESCRIPTION
**Pull request by MohitBis**
[CephAdm][Tier2]Configure RGW in warn state

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BZEVO7/

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
```
Comment (vamahaja) [inline suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml:1]: change suite name to `tier-2-rgw-with-warn-state.`

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+#   Add OSD nodes
+#   Perform object operations
+# ==================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+  - test:
```
Comment (vamahaja) [inline suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml:19]: Add extra line after each test case

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: bootstrap and configure RGW in warn state
+      destroy-cluster: false
+      polarion-id: CEPH-83573835
+      module: test_cephadm.py
+      name: Deploy cluster using cephadm and configure RGW in warn state.
+  - test:
```
Comment (vamahaja) [inline suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml:59]: Add comments for each test case. Include purpose of TC.

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml [hunk 4]
```diff
@@ -0,0 +1,122 @@
+---
+# ==================================================================================
```
Comment (vamahaja) [inline suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml:2]: Add same test suite for quincy

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml (hunk 1 above)
Comment (MohitBis) [reply suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml]: Thanks @vamahaja 
Updated

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml (hunk 2 above)
Comment (MohitBis) [reply suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml]: Thanks @vamahaja 
Updated

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml (hunk 3 above)
Comment (MohitBis) [reply suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml]: Thanks @vamahaja 
Updated

File: suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml (hunk 4 above)
Comment (MohitBis) [reply suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml]: Thanks @vamahaja 
Updated

**Comment by MohitBis** [comment]
Logs
Pacific: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BZEVO7/

Quincy: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AKJJ88/

**Review by vamahaja — CHANGES_REQUESTED** [review]
Add these suites for below metatadata files -
1. 5.3
2. 6.0
3. 6.1

File: suites/pacific/cephadm/tier-2-rgw-with-warn-state.yaml [hunk 5]
```diff
@@ -0,0 +1,138 @@
+---
+# ==================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_cephadm_conf_rgw_with_warn_state.yaml
```
[the remark this answers is not in the document]
Comment (MohitBis) [reply suites/pacific/cephadm/tier-2-rgw-with-warn-state.yaml]: Thanks @vamahaja 
Update

File: pipeline/metadata/5.3.yaml [hunk 6]
```diff
@@ -596,6 +596,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
           metadata:
             - rgw
+        - name: "Deploy cluster using cephadm and configure RGW in warn state"
```
Comment (vamahaja) [inline pipeline/metadata/5.3.yaml:599]: Add same tests for rhel 9 as well

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: pipeline/metadata/5.3.yaml (hunk 6 above)
Comment (MohitBis) [reply pipeline/metadata/5.3.yaml:599]: Thanks @vamahaja 
Updated

**Pull request merged**